Src/AutoFoqUnitTest/FixtureIntegrationTest.fs

```diff
@@ -66,3 +67,29 @@ let FixtureSuppliesValuesToAbstractGenericTypeWithNonDefaultConstructorWithMulti
     verify <@ not <| (Unchecked.defaultof<int> = result.Property1) @>
     verify <@ not <| (Unchecked.defaultof<int> = result.Property2) @>
     // Teardown
+
+type IInterface =
+    abstract ReturnSomething : unit -> string
+
+[<Fact>]
+let ``Fixture supplies return values for non-explicitly setup test-doubles`` () =
+    let sut = (Fixture().Customize(AutoFoqCustomization())).Create<IInterface>()
+    let actual = sut.ReturnSomething()
+    verify <@ not (isNull actual) @>
+
+[<Fact>]
+let ``Fixture customizations are propagated in Foq for non-explicitly setup test-doubles`` () =
+    let fixture = Fixture()
+    let expected = "Friday, 14 December 1984"
+    fixture.Register<string>(fun () -> expected)
+    fixture.ResidueCollectors.Add(

```

Comment:
> This happens already in that other test, above this one.

They aren't testing the same behaviour. The first one only checks to see if `actual` isn't null. This one is more explicit.